setup_data.sh

```diff
@@ -9,9 +9,19 @@ if [ -d tmp_rm_tools ]; then
 else
     git clone --depth 1 ${rm_tools_repo_url} tmp_rm_tools;
 fi;
+collection_exercise_host=${COLLECTION_EXERCISE_SERVICE_HOST:-localhost}
+collection_exercise_port=${COLLECTION_EXERCISE_SERVICE_PORT:-8145}
+collection_exercise_user=${SECURITY_USER_NAME:-admin}
+collection_exercise_password=${SECURITY_USER_PASSWORD:-secret}
```

Comment:
These variables will need to be added to the Jenkins file and possibly the config file.